src/kintone_client/user.cljc

```diff
@@ -220,3 +223,91 @@
                  (some? size) (assoc :size size))]
     (pt/-get conn (pt/-user-api-url conn path/group-users) {:params params})))
 
+(defn- get-all-by
+  [conn func query-key response-key values]
+  (go-loop [[parted :as rests] (partition-all 100 values)
+            ret []]
+    (if (empty? parted)
+      (t/->KintoneResponse {response-key ret} nil)
+      (let [res (<! (func conn {query-key parted}))]
+        (if (:err res)
+          res
+          (recur (rest rests) (concat ret (-> res :res response-key))))))))
+
+(defn- get-all
+  [conn func response-key]
+  (let [max-size 100]
+    (go-loop [ret []]
+      (let [res (<! (func conn {:offset (count ret)
+                                :size max-size}))]
+        (if (:err res)
+          res
+          (let [users (-> res :res response-key)
+                ret (concat ret users)]
+            (if (< (count users) max-size)
+              (t/->KintoneResponse {response-key ret} nil)
+              (recur ret))))))))
+
+(defn get-all-users
+  "Gets information of all users.
+
+  opts - If :ids and :codes are both set, an error will return. Only one can be set.
+
+    :ids - A list of User IDs.
+           sequence of integer
+
+    :codes - A list of User Codes (log-in names).
+             sequence of string"
+  [conn {:keys [ids codes]}]
+  (cond
+    (some? ids) (get-all-by conn get-users :ids :users ids)
+    (some? codes) (get-all-by conn get-users :codes :users codes)
+    :else (get-all conn get-users :users)))
```

Comment:
I'll add an additional signature like `(get-all-users conn)` later. It just falls back to the 2-arity signature.